.github/workflows/test.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11']
+        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11', '3.12']
```

Comment:
This is a tool supposed to run in CI. Why do we bother with anything than latest Python in the first place?

Reply:
My stance is: as long as it runs in those version, lets run it in those versions. People might need certain versions of python or whatever.